lib/sqlalchemy/ext/mutable.py

```diff
@@ -389,10 +389,11 @@ def __setstate__(self, state):
 from ..orm.decl_api import DeclarativeAttributeIntercept
 from ..orm.state import InstanceState
 from ..orm.unitofwork import UOWTransaction
-from ..sql.base import SchemaEventTarget
+from ..sql.base import _NoArg, NO_ARG, SchemaEventTarget
```

Comment:
**mike bayer** (zzzeek) wrote:

we have a new linter as of 5 minutes ago that will complain about this :).  if you can rebase to latest main you will want to split these into separate lines.

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/4479

Reply:
Latest push should solve this.